src/scripts/main.js

```diff
@@ -1,3 +1,361 @@
+          return b.children[2].innerText.localeCompare(a.children[2].innerText);
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Age':
+        sort = listTable.sort((a, b) => {
+          return b.children[3].innerText - a.children[3].innerText;
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Salary':
+        sort = listTable.sort((a, b) => {
+          return convertNum(b.children[4].innerText)
+            - convertNum(a.children[4].innerText);
+        });
+
+        table.append(...sort);
+    }
+  } else {
+    nameTitle = e.target.innerText;
+
+    switch (e.target.innerText) {
+      case 'Name':
+        sort = listTable.sort((a, b) => {
+          return a.children[0].innerText.localeCompare(b.children[0].innerText);
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Position':
+        sort = listTable.sort((a, b) => {
+          return a.children[1].innerText.localeCompare(b.children[1].innerText);
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Office':
+        sort = listTable.sort((a, b) => {
+          return a.children[2].innerText.localeCompare(b.children[2].innerText);
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Age':
+        sort = listTable.sort((a, b) => {
+          return a.children[3].innerText - b.children[3].innerText;
+        });
+
+        table.append(...sort);
+        break;
+
+      case 'Salary':
+        sort = listTable.sort((a, b) => {
+          return convertNum(a.children[4].innerText)
+            - convertNum(b.children[4].innerText);
+        });
+
+        table.append(...sort);
+    }
+  }
+});
+
+table.addEventListener('click', (e) => {
+  [...document.querySelectorAll('tr')].slice(1, -1).forEach((item) => {
+    item.classList = '';
+  });
+
+  e.path[1].className = 'active';
+});
+
+const massageResultTrue = Promise.resolve('new employer created');
+const massageResultFalse = Promise.reject(Error('some wrong, write correct'));
+
+document.querySelector('button').addEventListener('click', e => {
+  const div = document.createElement('div');
+
+  div.dataset.qa = 'notification';
+
+  e.preventDefault();
+
+  const dataInput = document.querySelectorAll('input');
+  const selectSity = document.querySelector('select');
+  const optionName = document.querySelectorAll('option');
+
+  let controlForm = true;
+
+  if (dataInput[0].value.length < 4 || +dataInput[0].value) {
+    controlForm = false;
+  }
+
+  if (dataInput[1].value.length < 4 || +dataInput[1].value) {
+    controlForm = false;
+  }
+
+  if (dataInput[2].value < 18) {
+    controlForm = false;
+  }
+
+  if (dataInput[3].value < 1) {
+    controlForm = false;
+  }
+
+  if (controlForm === false) {
+    massageResultFalse
+      .catch((error) => {
+        div.classList = ('notification error');
+        div.innerText = error;
+        document.body.append(div);
```

Comment:
![image](https://user-images.githubusercontent.com/86923089/205126933-1c236841-9146-40bf-8699-b63417b1cecd.png)
the message looks not good. append some title, using `h6` for example, (`Error`/`Success`) and message wrapped in `p` tag inside the `div`. `h6` and `p` have default styles